flatlaf-core/src/main/java/com/formdev/flatlaf/FlatLaf.java

```diff
@@ -36,20 +36,13 @@
 import java.util.function.Consumer;
 import java.util.logging.Level;
 import java.util.logging.Logger;
-import javax.swing.AbstractButton;
-import javax.swing.JLabel;
-import javax.swing.JRootPane;
-import javax.swing.JTabbedPane;
-import javax.swing.LookAndFeel;
-import javax.swing.PopupFactory;
-import javax.swing.SwingUtilities;
-import javax.swing.UIDefaults;
-import javax.swing.UIManager;
-import javax.swing.UnsupportedLookAndFeelException;
+import javax.swing.*;
```

Comment:
This is probably not desired.  Explicit imports is preferred.

Reply:
ok, I wait your codestyle
https://github.com/JFormDesigner/FlatLaf/issues/71

Reply:
Although .* imports seems like a code style issue.. Its not especially in the case of root level UI code like L&F.  This should be reverted.